code/model/DMSDocument.php

```diff
@@ -1341,14 +1359,32 @@ public function index(SS_HTTPRequest $request)
                         return $path;
                     }
 
+                    // set fallback if no config nor file-specific value
+                    $disposition = 'attachment';
+
+                    // file-specific setting
+                    if( $doc->DownloadBehavior == 'open' ){
```

Comment:
Please can you follow SS/PSR-2 coding standards. space before `(` and not after, space after `)` and not before, `elseif` and not `else if`